tomviz/SaveLoadStateReaction.cxx

```diff
@@ -79,6 +80,16 @@ bool SaveLoadStateReaction::loadState()
 
 bool SaveLoadStateReaction::loadState(const QString& filename)
 {
+  if (ModuleManager::instance().hasDataSources()) {
+    if (QMessageBox::No ==
+        QMessageBox::warning(
+          tomviz::mainWidget(), "Load State Warning",
+          "Current data and operators will be cleared when "
+          "loading a state file.  Proceed anyway?",
+          QMessageBox::Yes | QMessageBox::No, QMessageBox::No)) {
```

Comment:
@alesgenova Pointed out to me earlier today that ```QMessageBox::warning(...)``` are marked as obsolete and should not be used in new code. We doesn't have to do it in this PR, but we should move away from using them.

Reply:
Actually, it's only the signatures that take int's for the button types .... as you were.